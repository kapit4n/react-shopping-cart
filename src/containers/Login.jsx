import react from "react";
import LoginComponent from "../components/LoginComponent";

export default class Login extends react.Component {
  render() {
    return (<LoginComponent />)
  }
}
