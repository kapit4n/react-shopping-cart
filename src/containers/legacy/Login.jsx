import react from "react";
import LoginComponent from "../../components/legacy/LoginComponent";

export default class Login extends react.Component {
  render() {
    return (<LoginComponent />)
  }
}
