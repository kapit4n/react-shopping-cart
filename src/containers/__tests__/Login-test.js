import Login from '../Login'

import Enzyme, { shallow } from 'enzyme';
// Import Adapter
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
//use adapter
Enzyme.configure({ adapter: new Adapter() });

describe('shallow render', () => {
  const wrapper = shallow(<Login />)
  it('', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
