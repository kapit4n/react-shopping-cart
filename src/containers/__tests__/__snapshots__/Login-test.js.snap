// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow render  1`] = `ShallowWrapper {}`;
